WORKING_GROUPS.md

```diff
@@ -0,0 +1,203 @@
+# IPFS Project Working Groups
+
+## Index
+
+- [Description](#description)
+- [Working Group Lifecycle](#working-group-lifecycle)
+- [Current Working Groups](#current-working-groups)
+	- [Development and Maintenance Working Groups](#development-and-maintenance-working-groups)
+	- [Community, Education and Usage  Working Groups](#community-education-and-usage-working-groups)
+	- [Ongoing Efforts to Support Specific Uses](#ongoing-efforts-to-support-specific-uses)
+- [Other not yet classified as a working group](#other-not-yet-classified-as-a-working-group)
+
+
+## Description
+
+The IPFS Project Working Groups are teams of people that are appointed to research, develop and deploy work under the scope that the working group is focused. This structure is designed to provide clarity and direction to the project, enabling individual contributors to focus their time and energy on the areas they are most interested.
+
+Each Working Group has its own cadence, lifecycle and objectives.
+
+## Working Group Lifecycle
+
+TODO: Define how a working group gets created, evaled and how to decide if its purpuse has been fulfilled and should be dismantled
+
+## Current Working Groups
+
+### Development and Maintenance Working Groups
+
+- [Release](#release)
+- [Specifications](#specifications)
+- [Files (Unixfs), DEX and Data Structures on top of IPLD](#files-unixfs-dex-and-data-structures-on-top-of-ipld)
+- [Runtime Support and Build](#runtime-support-and-build)
+- [QA, Testing and Dev Team Enablement](#qa-testing-and-dev-team-enablement)
+- [Security](#security)
+- [Performance and Benchmarking](#performance-and-benchmarking)
+
+#### Release
+
+The Release Working Group manages the releases for go-ipfs, js-ipfs and its internal dependencies.
```

Comment:
Do you want this to be _one_ working group or do you want it to be separate release working groups for each language?

Reply:
I provided an answer on your previous comment, perhaps it got hidden with the rearranging. Here you go https://github.com/ipfs/ipfs/pull/285#discussion_r153806997

Reply:
> I considered this and I'm confident that releasing JS and Go can be handled by the same team. In the end, the Release team should focus on auditing the release, ensuring that the quality bar didn't lower (check features added, CI, documentation and so on), that communication with the community and partners happens (changelog, post, twitter, reddit, emails and so on).
> 
> Yes, there are some people that have more context of what is happening in Go and others in JS, but in the end it is the same protocol and given that this is a Working Group and not a Working individual, we can have 2~4 people that share enough context to be capable of launching the rocket with the same level of quality.

Reply:
Adding to the above, there should be some work on a release that involves checking spec compliance and interoperability.

Reply:
Defining the release roadmap and deciding what actually goes into a release feels like a responsibility of the dev team for that implementation, since they ultimately know when something is ready to be released or not.

With the rest of the points, I feel like we can fold that into QA, as it got to do with some sort of automated pipelines, testing and feedback cycles.